src/components/Tooltip/Tooltip.vue

```diff
@@ -1,28 +1,36 @@
 <template>
   <div class="relative">
     <div
-      class="absolute"
+      :class="['absolute', { 'hidden': !hover }]"
       :style="tooltipPositionStyle"
     >
       <div
         ref="tooltipContainer"
         :class="[
           'px-4 py-3 text-xs rounded-md m-auto bg-gray-700 text-white relative',
-          {'opacity-0': !hover}
+          { 'opacity-0': !hover },
```

Comment:
Nit: this is happening a lot throughout, where there seems to be a trailing comma in the class attributes. Not sure why lint isn't picking up on this particular use case. 🤔

Reply:
Found the rule! Fixed!